src/masternodes/mn_checks.cpp

```diff
@@ -2652,6 +2655,18 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
             auto attributes  = mnview.GetAttributes();
             attributes->time = time;
 
+            if (tokenId && token.IsLoanToken() && IsEVMEnabled(height, mnview, consensus)) {
```

Comment:
Validation check for CreateToken is already done above here, do we need to include tokenId here? the ResVal value is ensured to be non-null